src/pages/payment-methods/index.js

```diff
@@ -178,15 +182,28 @@ const PaymentMethods = () => {
                                                     <Th />
                                                 </Tr>
                                             </Thead>
-                                            <Tbody>
-                                                {pd.data.map((data, indx) => (
-                                                    <ExpandList
-                                                        key={indx}
-                                                        data={data}
-                                                        is_crypto={pd.is_crypto}
-                                                    />
-                                                ))}
-                                            </Tbody>
+                                            <TbodyContainer>
+                                                {(config) => (
```

Comment:
any reason why is this a function?, and when do we call it, if the intention is the check whether config is available, we can do 
```
{config && <Tbody></Tbody>}
```

Reply:
because we want to use `useContext` so we can access it on the child component of the layout.
since `PaymentMethods` component is actually layout we cannot access to `LocationContext` inside that so, I decided to create a Container then use the `useContext` there.